pydis_site/apps/api/models/log_entry.py

```diff
@@ -1,3 +1,5 @@
+import textwrap
```

Comment:
This is not being used.

Reply:
woops, forgot to run flake8 after moving to the django builtin stuff